stdpopsim/catalog/pongo_abelii.py

```diff
@@ -20,36 +20,36 @@
 # List of chromosomes.
 
 # length information can be found here
-# <http://https://www.ncbi.nlm.nih.gov/genome/325>
-# Orangutan Jan. 2018 (Susie_PABv2/ponAbe3) Browser Sequences
-# Note: no chrY
-# Average recombination rate estimated in Locke et al (doi:10.1038/nature09687)
-# which reports 0.95 +/- 0.72 cM/Mb
+# <http://https://www7.ncbi.nlm.nih.gov/genome/325>
```

Comment:
"http://" is duplicated here, and I had to remove the 7 from `www7` to get the orangutan page.

Reply:
Thanks Graham, not sure how that got mangled. Will fix.